app/serializers/api/v1/offer_serializer.rb

```diff
@@ -3,6 +3,8 @@ module Api::V1
   class OfferSerializer < ApplicationSerializer
     embed :ids, include: true
 
+    OFFER_TIME_ATTRIBUTES = Offer.columns_hash.select{|k,v| v.type == :datetime}.keys
```

Comment:
Unused block argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used.<br>Line is too long. [85/80]<br>Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.